### PR TITLE
fix(tests): better migration message functional tests

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes issues like https://circleci.com/gh/vladikoff/fxa-content-server/205

### CODE REVIEW

File: tests/functional/sync_sign_in.js [hunk 1]
```diff
@@ -109,8 +109,7 @@ define([
         .get(require.toUrl(PAGE_URL_WITH_MIGRATION))
         .setFindTimeout(intern.config.pageLoadTimeout)
         .execute(listenForFxaCommands)
-        .findByCssSelector('.info.nudge')
-        .isDisplayed()
```
Comment (vladikoff) [inline tests/functional/sync_sign_in.js]: `.info.nudge` shows up in the dom faster than it is displayed, need to use `visibleByQSA`

**Comment by vladikoff** [comment]
@shane-tomlinson @zaach r?

File: tests/functional/sync_sign_in.js (hunk 1 above)
Comment (philbooth) [inline tests/functional/sync_sign_in.js]: Oh, that one's my fault sorry. Will fix and PR.

Comment (vladikoff) [inline tests/functional/sync_sign_in.js]: > Oh, that one's my fault sorry. Will fix and PR.

This is already a PR ;)

Comment (philbooth) [inline tests/functional/sync_sign_in.js]: I meant add your suggested use of `visibleByQSA` and PR that. :)

Comment (philbooth) [inline tests/functional/sync_sign_in.js]: > I meant add your suggested use of visibleByQSA and PR that.

...and then I scrolled down. :blush:

**Pull request merged**